src/client/cli/formatter/json_formatter.cpp

```diff
@@ -63,29 +63,31 @@ std::string mp::JsonFormatter::format(const InfoReply& reply) const
 
     info_json.insert("errors", QJsonArray());
 
-    for (const auto& info : reply.info())
+    for (const auto& info : reply.detailed_report().details())
```

Comment:
Key-value pairs in JSON are automatically sorted in alphabetical order

Reply:
Hmm, you mean Qt does that, right? I think JSON itself is unordered (apart from arrays). Either way, we can't guarantee a specific ordering, so that's fine.